dbconnect.py

```diff
@@ -0,0 +1,72 @@
+#TODO Needs more work to make it complete
+
+import psycopg2
+
+
+class DBConnect(object):
+    """
+    Class for connecting to a postgresql database using a with statement
+    """
+
+    def __init__(self, dbhost='', db='', dbuser='', dbpass='', dbport=3306, autocommit=False,
+                 *args, **kwargs):
+        try:
+            self.conn = psycopg2.connect(host=dbhost, database=db, user=dbuser,
```

Comment:
any way to rework this to use a context manager?  (the with statement?)  That way connections are never left hanging open.